source/documentation/06-api-reference.md

```diff
@@ -134,6 +136,34 @@ These error codes provide more information about why a request failed.
 | General | P0920 | Request blocked by security rules | Our firewall blocked your request. See Troubleshooting section for details. |
 | General | P0999 | GOV.UK Pay is unavailable | The GOV.UK Pay service is temporarily down. |
 
+### API errors caused by payment statuses
+
+The API error codes in this section are driven by payment status.
+
+| Error code | Meaning | Cause |
+|:---|:---|:---|
+| P0050 | Payment provider returned an error | Multiple possible causes, for example a configuration problem with the payment provider, or incorrect login credentials; contact us, quoting the error code |
+| P0040 | Payment was cancelled by service | The service cancelled the payment |
+| P0010 | Payment method rejected | Multiple possible causes, for example the user does not have enough money in account, or 3D Secure authentication failure |
+| P0020 | Payment expired | A payment has not been confirmed and completed within one hour of being created; if the payment was already authorised, GOV.UK PAY will send a cancellation to the payment provider |
```

Comment:
“The payment was not confirmed…” might reflect the finality of the situation better

Reply:
Changed